geode-client-native/src/cppcache/impl/ThinClientRedundancyManager.hpp

```diff
@@ -56,7 +56,7 @@ namespace gemfire {
    void netDown( );
    void acquireRedundancyLock( ) { m_redundantEndpointsLock.acquire_read( ); }
    void releaseRedundancyLock( ) { m_redundantEndpointsLock.release( ); }
-   volatile bool allEndPointDiscon() { return m_IsAllEpDisCon;}
+   bool allEndPointDiscon() { return m_IsAllEpDisCon;} //Copyright: Amdocs Software Systems Limited, 2016
```

Comment:
It is just clarification that this particular line (in this case) was added by Amdocs Software System Limited.